### PR TITLE
Add meta description/keywords.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/420247669377092

Currently, this is a proof of concept demonstrating how. The only page currently involved is the "home" page. Once we have descriptions+keywords for the remaining pages, they'll be added to this PR.

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

File: index.adoc [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+description: >
```
Comment (Palleas) [inline index.adoc]: Do we need a title here also?

Comment (bbewan) [reply index.adoc]: If we don't specify `titletext`, the title tag gets the text from the page title.

**Comment by bbdocs** [comment]
See the website here: http://47-bb-docs.surge.sh

**Pull request merged**